core/src/main/resources/config.yml

```diff
@@ -142,6 +142,12 @@ cache-images: 0
 # Allows custom skulls to be displayed. Keeping them enabled may cause a performance decrease on older/weaker devices.
 allow-custom-skulls: true
 
+# The maximum number of custom skulls to be displayed per player. Increasing this may decrease performance on weaker devices.
+max-visible-custom-skulls: 128
```

Comment:
Perhaps add a -1 option where custom skulls are never culled?

Reply:
Added the option in the latest commits to disable the range and limit checks with -1.